resources/docker/default-config.xml

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<wfs xmlns="http://www.3dcitydb.org/importer-exporter/config" xmlns:ows="http://www.opengis.net/ows/1.1"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.3dcitydb.org/importer-exporter/config ../../src/main/webapp/WEB-INF/schemas/config/config.xsd">
```

Comment:
The path of schema location is not valid anymore once it has been copied to "src/main/webapp/WEB-INF" during the docker build. Maybe it doesn't matter.?

Reply:
No, it doesn't matter imho. We currently do not programmatically validate the `config.xml` when launching the WFS (well, maybe we should?). But even if would like to do so, I would not rely on the `schemaLocation` value but would rather read the schema from a fixed location. So, the `schemaLocation` is only relevant if you want to validate the file externally (for instance, when changing it manually with an XML editor). But I guess in the Docker scenario, a user would rather edit and mount his or her own `config.xml` file instead of the one inside the container.

Reply:
Agree. I think, we should support validating the `config.xml` when launching the WFS in future. That will allow avoiding unexpected errors caused by an invalid configuration during the runtime.